test/extended/image_ecosystem/s2i_php.go

```diff
@@ -19,7 +20,8 @@ var _ = g.Describe("[image_ecosystem][php][Slow] hot deploy for openshift php im
 		oc              = exutil.NewCLI("s2i-php", exutil.KubeConfigPath())
 		hotDeployParam  = "OPCACHE_REVALIDATE_FREQ=0"
 		modifyCommand   = []string{"sed", "-ie", `s/\$result\['c'\]/1337/`, "src/Template/Pages/home.ctp"}
-		pageCountFn     = func(count int) string { return fmt.Sprintf(`<span class="code" id="count-value">%d</span>`, count) }
+		pageRegexpCount = `<span class="code" id="count-value">([^0][0-9]*)</span>`
```

Comment:
i think this should be a `+` and not a `*` so it can't match an empty value.

Reply:
i guess this current regex is matching `>` followed by "anything other than 0" followed by N digits followed by `<`

so i guess making it a `+` wouldn't work, but the current regex will match something like `> <` too.  I guess it's not a big deal.

Reply:
Yeah http://www.rexegg.com/regex-quickstart.html would imply that as well ... empty could not be zero ... i.e. `[^0]`, and you then have zero or more 0-9 ...

Do you want me to change https://github.com/openshift/origin/blob/master/test/extended/image_ecosystem/sample_repos.go#L18 as well ?

Reply:
ah never mind .... was still crafting that update while you posted yours

btw when I tried it it did in fact fail